<launch>

  <node pkg="project3_phase1" type="control_node.py" name="controller" output="screen">
    <param name="linear_k_p" value="0.05"/>
    <param name="linear_k_d" value="0.5"/>
    <param name="linear_k_i" value="0.005"/>
    <param name="angular_k_p" value="10"/>
    <param name="angular_k_d" value="200"/>
    <param name="angular_k_i" value="0"/>
    <param name="dt" value="0.005"/>
    <param name="epsilon" value="0.01"/>
    <param name="iterations" value="5"/>
  </node>

  <node pkg="project3_phase1" type="mission_node.py" name="mission_node"  output="screen"></node>
  <node pkg="project3_phase1" type="monitor.py" name="monitor" ></node>

  <include file="$(find turtlebot3_gazebo)/launch/turtlebot3_empty_world.launch">
    <arg name="x_pos" value="0.0"/>
    <arg name="y_pos" value="0.0"/>
    <arg name="z_pos" value="0.0"/>

  </include>  

  <include file="$(find turtlebot3_gazebo)/launch/turtlebot3_gazebo_rviz.launch">
  </include>
</launch>